src/main.ts

```diff
@@ -47,7 +52,7 @@ const tryRedirect = async () => {
     setMessage('Not found: No Endorsement');
   } else {
     try {
-      const href = `https://${endorsedUI.value}.ipfs.cf-ipfs.com/${walletParam}`;
```

Comment:
`redirectParams.getAll` is a function, so this always returns true. Writing a function as suggested above will be easier to write correctly and more readable.

Reply:
functionized so hopefully that works